pkg/apis/compliance/v1alpha1/compliancesuite_types.go

```diff
@@ -175,3 +186,32 @@ func (s *ComplianceSuite) IsResultAvailable() bool {
 	result := s.LowestCommonResult()
 	return result != "" && result != ResultNotAvailable
 }
+
+// ApplyRemediations returns whether the ComplianceSuite requires
+// that the CoplianceRemediations that were generated from it be
+// applied.
+func (s *ComplianceSuite) ApplyRemediations() bool {
```

Comment:
nit: the name ApplyRemediations implies a direct action as such.. Might be clearer as HasApplyRemediations or similar.